src/i18n/translations.ts

```diff
@@ -453,7 +454,8 @@ export default {
         errorDescriptionAppIdNotSet: 'No application ID given. Please provide the ID as query parameter, e.g. ?applicationId=1909',
         errorDescriptionAppConfigNotFound: 'The application with ID {{applicationId}} could not be loaded correctly.',
         errorDescriptionAppConfigStaticNotFound: 'The configuration of the application could not be loaded correctly.',
-        permissionDeniedUnauthorized: 'This application is not public. Authentication required.'
+        permissionDeniedUnauthorized: 'This application is not public. Authentication required.',
+        rerouteToLoginPage: 'Return to login page.'
```

Comment:
Usually the error indicates that the user hasn't been on the login page yet. I would suggest to rename the text to something like `To Login page`

Reply:
Have made changes accordingly